src/version/version.go

```diff
@@ -0,0 +1,24 @@
+package version
+
+type RepositoryVersionControl interface {
+	GetMessage() (string, error)
+	GetVersionUpdateType(message string) (string, error)
+	UpgradeRemoteRepository() error
+}
+
+type FilesVersionControl interface {
+	UpgradeChangeLog(path string) error
+	UpgradeVariableInFiles(variableName string, filesList []string) error
+}
+
+type Version struct {
+	repoVersionControl  RepositoryVersionControl
+	filesVersionControl FilesVersionControl
+}
+
+func NewService(repoVersionControl RepositoryVersionControl, filesVersionControl FilesVersionControl) *Version {
```

Comment:
NewSemantic? @esequielvirtuoso

Reply:
I prefer to keep only `New` since the package is now called semantic.
So we can call it like `semantic.New()` instead of repeating the word `semantic` like `semantic.NewSemantic()`.